tests/baselines/reference/assignmentCompatOnNew.types

```diff
@@ -3,7 +3,7 @@ class Foo{};
 >Foo : Foo
 
 function bar(x: {new(): Foo;}){}
->bar : (x: new () => Foo) => void
+>bar : (x: {    new (): Foo;}) => void

```

Comment:
I see how the signature annotation changed but shouldnt it be `{ new (): Foo; }`  or `{new(): Foo;}` instead ? Why is there extra space before `new`

Reply:
I'm _pretty sure_, it's because our default formatting for object literals which don't have any emit flags set is newline-separated with appropriate indentation. I could set some emit flags to try to make the objects single-line; but I imagine that default is actually _better_ when we're not using the single line string writer that we use for `.types` baselines and error messages. (eg, quickinfo, declarations)

Reply:
Hmm even if its our default one wit wouldnt print it shouldnt start with these extra spaces no?

Reply:
If it actually emits as
```
{
    new (): Foo;
}
```
and the printer is simply eliding newlines (it is), it would be.